package.json

```diff
@@ -1,80 +1,53 @@
 {
-  "name": "ngx-wig",
-  "version": "1.2.7",
+  "name": "ngx-wig-app",
+  "version": "0.0.0",
   "scripts": {
-    "build": "gulp build",
-    "build:watch": "gulp",
-    "docs": "npm run docs:build",
-    "docs:build": "compodoc -p tsconfig.json -n ngx-wig -d docs --hideGenerator",
-    "docs:serve": "npm run docs:build -- -s",
-    "docs:watch": "npm run docs:build -- -s -w",
-    "lint": "tslint --type-check --project tsconfig.json src/**/*.ts",
-    "lite": "lite-server",
-    "playground:build": "tsc -p playground -w",
-    "playground": "concurrently \"npm run build:watch\" \"npm run playground:build\" \"npm run lite\"",
-    "test": "tsc && karma start"
+    "lib-build": "ng build ngx-wig",
+    "start": "(ng build ngx-wig --watch &) && sleep 10 && ng serve",
```

Comment:
Is additional `&` a typo?

Reply:
@bampakoa nope, that's for parallel run. We run lib build watch and leave it running. And meanwhile waiting 10 secs before running `ng serve`

Reply:
@yuriykuzin is this OS agnostic?